app/models/info_request/response_gatekeeper/base.rb

```diff
@@ -0,0 +1,22 @@
+# -*- encoding : utf-8 -*-
+class InfoRequest
+  module ResponseGatekeeper
+    class Base
+      attr_reader :info_request, :allow, :reason
+
+      def initialize(info_request)
+        @info_request = info_request
+        @allow = true
+        @reason = nil
+      end
+
+      def allow?(email)
+        allow
+      end
+
+      def rejection_action
+        info_request.handle_rejected_responses
+      end
+    end
+  end
+end
```

Comment:
Fixed in https://github.com/mysociety/alaveteli/commit/6b7fb04f9afb2546594b08cd34bbc672579367fb